client/src/components/pages/Home/index.js

```diff
@@ -1,14 +1,138 @@
-import React from 'react';
+import React, { Component } from 'react';
+import Input from '../../common/Input';
+import Avatar from '../../common/Avatar';
+import UserMenu from '../../common/UserMenu';
+import People from './People';
+import Messages from './Messages';
+import api from '../../../services/api';
+import { filter } from '../../assets/svgPaths';
 import './index.css';
-import UnderConstruction from '../UnderConstruction';
-
-const Home = () => {
-  return (
-    <>
-      Home
-      <UnderConstruction />
-    </>
-  );
-};
+
+class Home extends Component {
+  state = {
+    data: {},
+    search: '',
+    menu: false,
+    messages: [
+      {
+        id: 1,
+        avatar:
+          'https://images.unsplash.com/photo-1516756587022-7891ad56a8cd?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60',
+        username: 'ammodaa',
+        lastMessage: 'Hey, what happened?',
+        messageTime: '19:14',
+      },
+      {
+        id: 2,
+        avatar:
+          'https://images.unsplash.com/photo-1531251445707-1f000e1e87d0?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60',
+        username: 'fadi',
+        lastMessage: 'Hey, what happened?',
+        messageTime: '19:14',
+      },
+    ],
+    showMessages: false,
+    showPeople: true,
+  };
+
+  componentDidMount() {
+    api.userNativeLang().then(res => this.setState({ data: res }));
```

Comment:
just something to think of, would be good to show some message if this fails (ie. have a catch), but don't waste time with it for now (you can create an issue to add it later if you want)

Reply:
Okay we'll add it later